app/assets/javascripts/application.js

```diff
@@ -229,28 +229,32 @@ function filter_by_level(item){
 
   // Note that class names don't map to log level names (label-info == notice)
   if(level == 'info'){
-    $('.label-info').closest('tr').show();
-    $('.label-default').closest('tr').show();
-    $('.label-warning').closest('tr').show();
-    $('.label-danger').closest('tr').show();
+    $('.label-info.result-filter-tag').closest('tr').show();
+    $('.label-default.result-filter-tag').closest('tr').show();
```

Comment:
@xprazak2 ,
You have applied a separate class to these label tags in `app/helpers/reports_helper.rb`.
Instead of individual `.label-{tag}.result-filter-tag` selector, only `.result-filter-tag` selector should work here.
i.e $('.result-filter-tag').closest('tr').show();